python/cli/prelude_cli/views/iam.py

```diff
@@ -24,8 +24,9 @@ def register_account(controller):
     email = click.prompt('Enter your email')
     name = click.prompt('(Optional) Enter your name', default='', show_default=False)
     company = click.prompt('(Optional) Enter your associated company', default='', show_default=False)
+    slug = click.prompt('(Optional) Enter an account slug', default='', show_default=False)
```

Comment:
'' here but none in click option.

Reply:
`click.prompt` needs an input (it'll just keep asking the same question if I default None). Ultimately, empty string and None will be treated the same way in sdk (`if slug: <blah>`)